Tasks/VSMobileCenterTest/task.json

```diff
@@ -0,0 +1,354 @@
+{
+    "id": "AD5CD22A-BE4E-48BB-ADCE-181A32432DA5",
+    "name": "VSMobileCenterTest",
+    "friendlyName": "Visual Studio Mobile Center Test",
+    "description": "Test mobile app packages with Visual Studio Mobile Center",
+    "helpMarkDown": "Test mobile app packages to Visual Studio Mobile Center",
+    "category": "Test",
+    "visibility": [
+        "Build"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 111,
+        "Patch": 0
+    },
+    "groups": [
+        {
+            "name": "prepare",
+            "displayName": "Prepare Test",
+            "isExpanded": true
+        },
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": true 
+        }
+    ],
+    "inputs": [
+        {
+            "name": "credsType",
+            "type": "pickList",
+            "label": "Authentication Method",
+            "defaultValue": "serviceEndpoint",
+            "required": true,
+            "helpMarkDown": "Use Mobile Center service endpoint connection or enter credentials to connect to Visual Studio Mobile Center.",
+            "options": {
+                "serviceEndpoint": "Mobile Center Connection",
+                "inputs": "Credentials"
+            }
+        },
+        {
+            "name": "serverEndpoint",
+            "type": "connectedService:vsmobilecenter",
+            "label": "Mobile Center Connection",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select the service endpoint for your Visual Studio Mobile Center connection. To create one, click the Manage link and create a new service endpoint.",
+            "visibleRule": "credsType = serviceEndpoint"
+        },
+        {
+            "name": "username",
+            "type": "string",
+            "label": "Mobile Center Username",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Visit https://mobile.azure.com/settings/profile to get your username.",
+            "visibleRule": "credsType = inputs"
+        },
+        {
+            "name": "password",
+            "type": "string",
+            "label": "Mobile Center Password",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Visit https://mobile.azure.com/settings/profile to set your password.",
```

Comment:
Add a note here to use a secret build variable for the password. We have similar text in other tasks that you can follow.